lib/hashes.md

```diff
@@ -5,40 +5,40 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A Hash is a collection of data elements that are indexed by a key that is another data element.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+A Hash would be used to organize data into pairs, such as names and addresses or cities and states.
 
 # How are Hashes and Arrays similar? How are they different?
 
-Replace me with your response...
+Both are collections of data elements, but Arrays are indexed by sequential integers, while Hashes are indexed by other data elements.
 
 # How do you retrieve a particular value from a Hash?
 
-Replace me with your response...
+By using the data element associated with that value as the index key.
```

Comment:
It's hard for me to picture what you're trying to convey here. Using an example might help clarify.